test/api.js

```diff
@@ -314,9 +314,14 @@ test('test file in node_modules is ignored', function (t) {
 test('Node.js-style --require CLI argument', function (t) {
 	t.plan(1);
 
+	var requirePath = path.relative(process.cwd(), path.join(__dirname, 'fixture', 'install-global.js'));
+	// `path` will use \\ to seperate paths on windows, but require doesn't.
+	requirePath = requirePath.split(path.sep).join('/');
```

Comment:
Shouldn't windows users be allowed to use `\` as a path separator, since that is what they are used to?

Seems to me that 

```
ava --require .\fixtures\install-global.js .\fixture\validate-installed-global.js
```

is the valid windows style alternative to 

```
ava --require ./fixtures/install-global.js ./fixture/validate-installed-global.js
```

I would prefer it if AVA accepted either pass separator and just normalized it internally.